__tests__/app.test.js

```diff
@@ -271,3 +272,32 @@ describe('PATCH /api/articles/:article_id', () => {
       });
   });
 });
+describe('DELETE /api/comments/:comment_id', () => {
+  test('204: should delete the given comment by comment_id', () => {
+    return request(app)
+      .delete('/api/comments/3')
+      .expect(204)
+      .then(() => {
+        return db.query('SELECT * FROM comments WHERE comment_id = 3');
+      })
+      .then(({ rows }) => {
+        expect(rows).toHaveLength(0);
+      });
+  });
```

Comment:
This is a nice test, but you dont need to have a query directly to the datbase like this. This is because this test, combined with the next two error handling tests should give you the confidence that this works as intended